test/unit/io_managers_test.py

```diff
@@ -249,3 +255,232 @@ def test_error_when_reading_view_without_metadata(pudl_sqlite_io_manager_fixture
+                            {
+                                "name": "str_factoid",
+                                "type": "string",
+                            },
+                        ],
+                        "primary_key": [
+                            "entity_id",
+                            "filing_name",
+                            "publication_time",
+                            "date",
+                            "utility_type_axis",
+                        ],
+                    },
+                }
+            ],
+        }
+    )
+
+    datapackage_path = tmp_path / "test_db_datapackage.json"
+    with datapackage_path.open("w") as f:
+        f.write(datapackage)
+
+    df = pd.DataFrame.from_records(
+        [
+            {
+                "entity_id": "C000001",
+                "utility_type_axis": "electric",
+                "filing_name": "Utility_Co_0001",
+                "date": datetime.date(2021, 12, 31),
+                "publication_time": datetime.datetime(2022, 2, 1, 0, 0, 0),
+                "str_factoid": "original 2021 EOY value",
+            },
+            {
+                "entity_id": "C000001",
+                "utility_type_axis": "electric",
+                "filing_name": "Utility_Co_0002",
+                "date": datetime.date(2021, 12, 31),
+                "publication_time": datetime.datetime(2022, 2, 1, 1, 1, 1),
+                "str_factoid": "updated 2021 EOY value",
+            },
+        ]
+    )
+
+    id_table = pd.DataFrame.from_records(
+        [
+            {"filing_name": "Utility_Co_0001", "report_year": 2021},
+            {"filing_name": "Utility_Co_0002", "report_year": 2021},
+        ]
+    )
+
+    conn = sa.create_engine(f"sqlite:///{db_path}")
+    df.to_sql("test_table_instant", conn)
+    id_table.to_sql("identification_001_duration", conn)
+
+    input_context = build_input_context(
+        asset_key=AssetKey("test_table_instant"),
+        resources={
+            "dataset_settings": mocker.MagicMock(
+                ferc1=mocker.MagicMock(xbrl_years=[2021])
+            )
+        },
+    )
+    io_manager = FercXBRLSQLiteIOManager(base_dir=tmp_path, db_name="test_db")
+    observed_table = io_manager.load_input(input_context)
+
+    assert len(observed_table) == 1
+    assert observed_table.str_factoid.to_numpy().item() == "updated 2021 EOY value"
+
+
+example_schema = pa.DataFrameSchema(
+    {
+        "entity_id": pa.Column(str, nullable=False),
+        "date": pa.Column("datetime64[ns]", nullable=False),
+        "utility_type": pa.Column(
+            str, pa.Check.isin(["electric", "gas", "total", "other"]), nullable=False
+        ),
+        "publication_time": pa.Column("datetime64[ns]", nullable=False),
+        "int_factoid": pa.Column(int),
+        "float_factoid": pa.Column(float),
+        "str_factoid": pa.Column("str"),
+    }
+)
+
+
+@hypothesis.given(example_schema.strategy(size=3))
+def test_get_unique_row_per_context(df):
```

Comment:
More context context... I'm not clear on what that means in XBRL-land.

And maybe add a note indicating what other tests in here this duplicates so we don't get confused later about why things show up twice and wonder if they're trying to test different things.  Or are you thinking of removing one or the other of them?

Reply:
I was thinking of removing the manual one.

Reply:
I think that's a fine idea.